### PR TITLE
Added back to back checkbox to the bar chart dialog

### LINKED ISSUE #6525: Add back-to-back bar charts to dialogue
This is one of three major improvements to the bar chart dialogue.

image

Population pyramids are an example of "back-to-back" bar charts.  Other examples are also shown in the figure.  I think they would be easier to do in situations where the left-hand side could just be negative, and this would perhaps be an option.  But good to have the possibility of both sides being positive as in the examples shown.

Ideally this would just be a checkbox labelled "Back to Back", but it may have to be more complicated than that.  It is only enabled when the optional factor is completed.  This second factor presumably can only have 2 levels.

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #6525
@africanmathsinitiative/developers @rdstern this is ready for review. Thanks.

### CODE REVIEW

File: instat/static/InstatObject/R/Rsetup.R [hunk 1]
```diff
@@ -42,7 +42,7 @@ packs <- c("abind", "agricolae", "agridat",
            "hydroGOF", "hydroTSM", "imputeTS", "ini", 
            "insight", "intervals", "isoband", "jpeg", 
            "jquerylib", "jsonlite", "Kendall", "KernSmooth", 
-           "keyring", "klaR", "knitr", "labeling", 
+           "keyring", "klaR", "knitr", "labeling", "lemon",
```
Comment (shadrackkibet) [inline instat/static/InstatObject/R/Rsetup.R:45]: This should not be edited manually. See the comment at the beginning of this file.

**Review by shadrackkibet — CHANGES_REQUESTED** [review]
Please resolve the conflicts in this PR so that others can review this PR.

File: instat/static/InstatObject/R/Rsetup.R (hunk 1 above)
Comment (shadrackkibet) [reply instat/static/InstatObject/R/Rsetup.R:45]: If you would like this to be added as a new package then add/open an issue for it. Also, make sure you link to this PR.

Comment (N-thony) [reply instat/static/InstatObject/R/Rsetup.R]: @shadrackkibet I think this is resolved. Please, have a look. Thanks.

**Review by rdstern — CHANGES_REQUESTED** [review]
@N-thony I hoped to check, but there were build errors

**Pull request closed without merging**